### DISCUSSION TITLE
Saving user settings

### DISCUSSION THREAD

**Discussion by nipunarora** [gitpod-io/openvscode-server#303] [Q&A]
This is a newbie question - but for openvscode-server, how can we save user settings across restarts? For instance plugins installed or keybindings changed etc.?

**Comment by akosyakov** [comment]
If it is the same machine and the same origin then it should work out of the box as far as I remember.

Later maybe we could hook it with Gitpod auto sync service, hooking with GitHub and MS is not an option as far as I know.
